styles/support-style.css

```diff
@@ -0,0 +1,182 @@
+:root {
+  --color-one: rgb(28, 34, 148);
+  --color-two: #e5e5e5;
+}
+
+p {
+  font-family: "Space Mono", monospace, sans-serif;
+}
+
+h1,
+h2,
+h3 {
+  font-family: "Work Sans", sans-serif;
+}
+
+/* navbar */
+.image {
+  height: 50px;
+}
+
+.navbar-light .navbar-toggler-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3e%3cpath stroke='rgba%2858, 78, 213, 1%29' stroke-linecap='round' stroke-miterlimit='10' stroke-width='4' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e");
+}
+
+.navbar-light .navbar-toggler {
+  color: transparent;
+  border: transparent;
+}
+
+.navbar-nav {
+  font-family: sans-serif;
+  font-size: 70%;
+  font-weight: lighter;
+}
+
+.nav-item a:hover {
+  text-decoration: underline solid 3px rgb(58, 78, 202);
+}
+
+.nav-item button {
+  background-color: rgb(58, 78, 202);
+  font-size: small;
+  font-weight: bold;
+}
+
+.navbar-toggler-icon:hover {
+  background-color: rgb(209, 208, 208);
+}
+
+/*  section-1 */
+.partner-main {
+  display: flex;
+  justify-content: center;
+  padding: 0 20px;
+  position: relative;
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  -webkit-box-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  align-items: center;
+  width: 100%;
+  max-width: 1280px;
+  padding: 10px 0;
+  margin: 0 auto;
+}
+
+.container-1 {
+  width: 100%;
+  display: flex;
+  margin: 50px 0;
+}
+
+.text {
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  padding: 0 50px 0 0;
+  width: 50%;
+  order: 1;
+}
+
+.heading-1 {
+  font-size: 36px;
+  line-height: 120%;
+  margin-bottom: 20px;
+  text-transform: uppercase;
+  font-family: "Work Sans", sans-serif;
+  font-weight: 700;
+  color: rgb(74, 80, 187);
+  text-align: left;
+  max-width: none;
```

Comment:
You usually use a value for `max-width`; if the value is `none` you can just remove it.

Reply:
@bermarte Thank you so much for your clarifications, i'll take them in consideration👍